testing/endtoend/components/validator.go

```diff
@@ -258,7 +258,17 @@ func (v *ValidatorNode) Start(ctx context.Context) error {
 		// See: https://docs.teku.consensys.net/en/latest/HowTo/External-Signer/Use-External-Signer/
 		args = append(args,
 			fmt.Sprintf("--%s=http://localhost:%d", flags.Web3SignerURLFlag.Name, Web3RemoteSignerPort),
-			fmt.Sprintf("--%s=%s", flags.Web3SignerPublicValidatorKeysFlag.Name, strings.Join(validatorHexPubKeys, ",")))
+		)
+		if v.config.UsePersistentKeyFile {
+			testNetDir := e2e.TestParams.TestPath + fmt.Sprintf("/proposer-settings/validator_%d", index)
```

Comment:
Why not something like:
```go
testNetDir := fmt.Sprintf("%s/proposer-settings/validator_%d", e2e.TestParams.TestPath, index)
```

Reply:
Also, this won't probably work on Windows right?
Do we want E2E tests to run on Windows?

Reply:
I don't think E2E can run on windows as it runs through bazel, i don't think bazel works properly on windows